src/plugins/action_change_status/src/changeStatus.ts

```diff
@@ -0,0 +1,86 @@
+import { ActionPluginOutput } from '../../types'
+
+type Status =
+  | 'Draft'
+  | 'Withdrawn'
+  | 'Submitted'
+  | 'Changes Required'
+  | 'Re-submitted'
+  | 'Completed'
```

Comment:
Consider moving this to the `types` file

Reply:
Done. :)